index.html

```diff
@@ -0,0 +1,201 @@
+<!doctype html>
+<script src=/boot.js></script>
+<title>Using Specifications</title>
+
+<div id=view>
+<main markdown>
+<header>
+  <h1 markdown>The `@using` Directive</h1>
+  <h2>for declaring specifications used by a schema</h2>
+
+  <table class=spec-md>
+    <tr>
+      <td>status</td>
+      <td id=spec-md-status>Draft</dd>
+    </tr>
+    <tr>
+      <td>version</td>
+      <td id=spec-md-version>0.1</dd>
+    </tr>
+    <tr>
+      <td>authors</td>
+      <td id=spec-md-authors>
+        <ul class=spec-authors-list>
+          <li><a rel=author href=mailto:ashi@apollographql.com>Ashi Krishnan</a>
+        </ul>
+      </td>
+    </tr>
+  </table>
+</header>
+
+## Problem
+
+Some specifications—such as the [federation specification](../federation-spec) and [CSDL specification](./csdl)—apply to GraphQL schema documents. They may place additional requirements on conforming documents. Such specifications may also introduce new types, directives, and other schema objects which can be used within those documents.
+
+The [GraphQL Schema Definition Language](https://spec.graphql.org/) does not provide a way to declare that a schema document implements additional specs beyond the GraphQL SDL. Currently, this information is derived from context, or through some explicit, out-of-document paramenter provided to a spec processor. As a consequence, given only the text of an SDL document, there is no reliable way to discern what specifications—and what version of those specifications—the document conforms to.
```

Comment:
When talking about schemas, we should reference the definition of SDL in the CSDL spec here (assuming that's where the definition will go).